app/frontend/wallet/shelley-wallet.ts

```diff
@@ -295,61 +288,75 @@ const ShelleyWallet = ({config, randomInputSeed, randomChangeSeed, cryptoProvide
   }
 
   async function getMaxNonStakingAmount(address) {
-    const utxos = (await getUTxOs()).filter(isUtxoNonStaking)
+    const utxos = (await getUTxOs()).filter(({address}) => !isGroup(address))
     return _getMaxSendableAmount(utxos, address, false, 0, false)
   }
 
-  const uTxOTxPlanner = async (args, txType) => {
-    const {address, coins, donationAmount, pools} = args
-    const utxoFilters = {
-      delegation: isShelleyUtxo,
-      nonStakingConversion: isUtxoNonStaking,
-      utxo: () => true,
-    }
-    const utxoFilter = utxoFilters[txType]
-    const accountAddress = await myAddresses.accountAddrManager._deriveAddress(0)
-    const availableUtxos = (await getUTxOs()).filter(utxoFilter)
+  type utXoArgs = {
```

Comment:
I find those alternating lower and upper case letters in `utxo` in the variable name confusing and it differs a few lines below in `uTxOTxPlanner` function name. I would rather coin the `utxo` as a single name and not try to match the words that it stands for, because it complicates camel casing as can be seen here

Reply:
You are right, fixed.